tests/component.test.js

```diff
@@ -0,0 +1,163 @@
+const ParchKey = require('../model/parch-key.js');
+const Component = require('../model/component.js');
+const Port = require('../model/port.js');
+const Coord = require('../model/coord.js');
+
+//Suppress console logs
+console.log = jest.fn();
+
+var l1 = 'layer-1-id';
+var l2 = 'layer-2-id';
+var p1 = new Port('port-1-label', l1, new Coord(0, 0));
+var p2 = new Port('port-2-label', l2, new Coord(0, 5));
```

Comment:
I think since these are going to be used throughout this file you might want to give them better names like `layer1`, `layer2`, `port_zero_zero`, `port_zero_five` or something like that

Reply:
They've been changed.